frontend/apps/reader/modules/readerstyletweak.lua

```diff
@@ -726,6 +727,7 @@ function ReaderStyleTweak:onToggleStyleTweak(tweak_id, item)
         else
             self.doc_tweaks[tweak_id] = nil
         end
+        text = _("Disabled style tweak: ")
```

Comment:
@Frenzie will confirm if there is a risk having translatable strings ending with a space - and translator missing it in their translation.

(These notifications will probably be long and end up truncated.)

Reply:
Yes, at a glance it'd be better to use the template function right here and in the else with %1 and `self.tweaks_in_dispatcher[tweak_id]` as an argument.